scripts/pi-hole/php/get.php

```diff
@@ -20,7 +20,12 @@
 function getTableContent($listname) {
 	global $db;
 	$entries = array();
-	$results = $db->query("SELECT * FROM $listname");
+	$querystr = implode(" ",array("SELECT ${listname}.*,\"group\".enabled as group_enabled",
+	                              "FROM $listname",
+	                              "LEFT JOIN ${listname}_by_group ON ${listname}_by_group.whitelist_id = ${listname}.id",
```

Comment:
This is hardcoding `whitelist_id`, which causes the blacklist pages to not show any domains.